frontend/src/metabase/collections/components/CollectionHeader/CollectionTimeline.tsx

```diff
@@ -16,13 +16,11 @@ const CollectionTimeline = ({
   const url = Urls.timelinesInCollection(collection);
 
   return (
-    <Tooltip tooltip={t`Events`}>
-      <Link to={url}>
-        <IconWrapper>
-          <Icon name="calendar" size={20} />
-        </IconWrapper>
-      </Link>
-    </Tooltip>
+    <Link to={url} tabIndex={-1}>
+      <Tooltip tooltip={t`Events`}>
+        <CollectionHeaderButton icon="calendar" iconSize={20} onlyIcon />
```

Comment:
Guess there is no need to pass `onlyIcon` prop here if it's already defined as a default one for `CollectionHeaderButton`?

Reply:
`iconSize` can also be a default prop. Or we can remove it from `CollectionHeaderButton's` `defaultProps` and pass it explicitly as it is now

Reply:
Ahhh yeah, good catch!